examples/nvme/perf/perf.c

```diff
@@ -1023,6 +1036,36 @@ associate_workers_with_ns(void)
 	return 0;
 }
 
+static void
+dissociate_worker_from_ns(void)
+{
+	struct ns_entry         *entry = g_namespaces;
+	struct worker_thread    *worker = g_workers;
+	struct ns_worker_ctx    *ns_ctx;
+
+	if (worker == NULL) {
+		return;
+	}
+	ns_ctx = worker->ns_ctx;
```

Comment:
I think this needs to be done inside the while (worker) loop - each worker may have one or more ns_ctx associated with it.